tests/common.go

```diff
@@ -4276,6 +4289,20 @@ func CreateS3BackupLocation(name string, uid, cloudCred string, cloudCredUID str
 
 // CreateS3BackupLocationWithContext creates backup location for S3 using the given context
 func CreateS3BackupLocationWithContext(name string, uid, cloudCred string, cloudCredUID string, bucketName string, orgID string, encryptionKey string, ctx context1.Context) error {
+	// check for cloud cred status before creation of backup location
+	cloudCredStatus := func() (interface{}, bool, error) {
+		status, err := IsCloudCredPresent(cloudCred, ctx, orgID)
+		if err != nil {
+			return "", true, fmt.Errorf("cloud cred %s present with error %v", cloudCred, err)
+		}
+		if status {
+			return "", true, nil
+		}
+		return "", false, nil
+	}
+	_, err := task.DoRetryWithTimeout(cloudCredStatus, defaultTimeout, defaultRetryInterval)
+	Inst().Dash.VerifySafely(err, nil, fmt.Sprintf("Verfiying cloud cred %s is present", cloudCred))
```

Comment:
I feel this should be done while creating cloud credential. @mkoppal-px

Reply:
I also think that would be better instead of checking status for all type of location.  Moved the code to CreateCloudCredential.